app/generators/questionnaire_generator.rb

```diff
@@ -172,6 +174,47 @@ def radio_option_body(question_id, option)
       option_body_wrap(question_id, option[:title], option[:tooltip], wrapped_tag)
     end
 
+    def generate_time(question)
+      body = time_body(question)
+      safe_join([content_tag(:p, question[:title].html_safe, class: 'flow-text'), body])
+    end
+
+    def time_body(question)
+      from = question[:hours_from] || 0
+      to = question[:hours_to] || 60
+      step = question[:hours_step] || 15
+
+      hours = time_dropdown(question[:id], from, to, step, 'Uren')
+      minutes = time_dropdown(question[:id], 0, 60, 15, 'Minuten')
+
+      safe_join([hours, minutes])
+    end
+
+    def time_dropdown(question_id, from, to, step, label)
+      elem_id = idify(question_id, label)
+      options = generate_dropdown((from...to).step(step), elem_id)
+      options = safe_join([
+                            options,
+                            content_tag(:label, label)
+                          ])
+      content_tag(:div, options, class: "input-field col s1 #{elem_id}")
+    end
+
+    def generate_dropdown(items, id, initial_option = nil)
+      body = []
+      if initial_option.present?
+        body << content_tag(:option, initial_option,
+                            value: '',
+                            selected: '',
+                            disabled: '')
+      end
```

Comment:
Dit roep je nooit aan in je code toch? Kun je ook weghalen/commenten als we het niet gebruiken. Dan blijft onze code coverage ook mooi op 100%.